scalding-core/src/main/scala/com/twitter/scalding/CascadingTokenUpdater.scala

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2014 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package com.twitter.scalding
+
+object CascadingTokenUpdater {
+  private final val lowestAllowed = 128 // cascading rules
+
+  // Take a cascading string of tokens and turns it into a map
+  // from token index to class
+  private def parseTokens(tokClass: String): Map[Int, String] =
+    if (tokClass == null || tokClass.isEmpty)
+      Map[Int, String]()
+    else
+      tokClass
+        .split(",")
```

Comment:
No idea, i guess we've been running with this for a while ok. You mean it includes blank entries in the event of repeated commas? We could filter them out here too